packages/website/assets/illustrations/cross.js

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore
```

Comment:
nit: The "Component per image" pattern here leads to *a lot* of trivial components... could a build step create them or could we re-use a single component... or just an <img> tag in the parent?